sdk/servicebus/Azure.Messaging.ServiceBus/TROUBLESHOOTING.md

```diff
@@ -226,7 +226,27 @@ When attempting to do a batch receive, i.e. passing a `maxMessages` value of 2 o
 
 ### Message or session lock is lost before lock expiration time
 
-The Service Bus service leverages the AMQP protocol, which is stateful. Due to the nature of the protocol, if the link that connects the client and the service is detached after a message is received, but before the message is settled, the message is not able to be settled on reconnecting the link. Links can be detached due to a short-term transient network failure, a network outage, or due to the service enforced 10-minute idle timeout. The reconnection of the link happens automatically as a part of any operation that requires the link, i.e. settling or receiving messages. Because of this, you may encounter `ServiceBusException` with `Reason` of `MessageLockLost` or `SessionLockLost` even if the lock expiration time has not yet passed. 
+A Service Bus queue or topic subscription has a lock duration set at the resource level. When the receiver client pulls a message from the resource, the Service Bus broker applies an initial lock to the message. The initial lock lasts for the lock duration set at the resource level. If the message lock isn't renewed before it expires, then the Service Bus broker releases the message to make it available for other receivers. If the application tries to complete or abandon a message after the lock expiration, the call fails.
+
+Service Bus leverages the AMQP protocol, which is stateful. Due to the nature of the protocol, if the link that connects the client and the service is detached after a message is received, but before the message is settled, the message is not able to be settled on reconnecting the link. Links can be detached due to a short-term transient network failure, a network outage, or due to the service enforced 10-minute idle timeout. The reconnection of the link happens automatically as a part of any operation that requires the link, i.e. settling or receiving messages. Because of this, you may encounter `ServiceBusException` with `Reason` of `MessageLockLost` or `SessionLockLost` even if the lock expiration time has not yet passed. 
```

Comment:
My understanding is that if the receiver-link is identified as detached (disposed) by the time application attempt to settle or there happens to be a new receiver-link, then library will be falling back to settle using connection level management-link ([java src](https://github.com/Azure/azure-sdk-for-java/blob/2186a4c341788e67df8a7b085266b96577e9770f/sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClient.java#L1634)).  Unsure if this is the same case for C#, but thought of sharing.

Reply:
It's the same fallback path in the .NET library, though that didn't exist when the TSG was first written.   I considered updating this, but I think trying to explain the details around fallback and that it is mostly successful but with really low throttles was too much detail.    I'm going to leave the original guidance for now and leave out the fallback details.   
_(rumor has it that they may deprecate the management fallback when durable terminus is available)_